### PR TITLE
New signup form

### PR DESCRIPTION
**Pull request by sara-fs**
# Description

Summary of the change and/or issue that is fixed.

Changes:

-   fixes issue with item
-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Refactor code
-   [ ] Update translation text
-   [ ] Breaking change
-   [ ] Dependency update
-   [ ] This change requires a documentation update
-   [ ] Release

### CODE REVIEW

**Comment by arash-fs** [comment]
Please check these Container background color (I think the color is different from the zeplin)

File: src/pages/signup/index.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    * {
+        margin-right: 1.6rem;
+        width: fit-content;
+    }
+    :last-child {
+        margin-bottom: 0;
+    }
+`
+
+const StyledText = styled(Text)`
```
Comment (arash-fs) [inline src/pages/signup/index.js]: Please add opacity 0.56 based on the design

File: src/components/custom/_signup-new.js [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+import PropTypes from 'prop-types'
+import { Input, Button } from 'components/form'
+import { FlexGridContainer } from 'components/containers'
+import { Header, Text, Checkbox } from 'components/elements'
+import device from 'themes/device.js'
+import Facebook from 'images/svg/facebook-blue.svg'
+import Google from 'images/svg/google.svg'
+import { localize, Localize, LocalizedLink } from 'components/localization'
+
+const SignupContent = styled.div`
```
Comment (arash-fs) [inline src/components/custom/_signup-new.js]: Please add border-radius and shadow based on the Zeplin.

File: src/components/custom/_signup-new.js [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+    padding: 1.6rem 1.6rem;
+    margin-top: 1.6rem;
+    border-radius: 0.6rem;
+    background-color: var(--color-grey-8);
+    border: solid 1px var(--color-grey-11);
+`
+const InputGroup = styled.div`
+    position: relative;
+    width: 100%;
+    margin: var(--text-size-m) 0;
```
Comment (arash-fs) [inline src/components/custom/_signup-new.js]: I think we should remove `0` in `margin` property here.

Comment (arash-fs) [inline src/components/custom/_signup-new.js]: Please check the `margin` value here, I think it's not based on our convention.

**Comment by arash-fs** [comment]
Please check the signup page background-color too, I think it's a little bit different from the design.

File: src/pages/signup/index.js [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+`
+
+const Line = styled.div`
+    width: 100%;
+    height: 1px;
+    background-color: var(--color-red);
+    margin-top: 2.4rem;
+`
+
+const NewSignup = () => {
```
Comment (oskar-binary) [inline src/pages/signup/index.js]: Please add the SEO component (and  add the document title) to this page

File: src/pages/signup/index.js [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+                        <Line />
+                        <StyledText color="grey-12">
+                            {localize(
+                                'By giving us your email address, you agree to receive marketing emails from us. You may unsubscribe from these emails at any time.',
+                            )}
+                        </StyledText>
+                    </Content>
+                )}
+
+                <Signup
```
Comment (oskar-binary) [inline src/pages/signup/index.js]: can we set the focus on the input after the component has been mounted too please 🙏

**Pull request merged**